### PR TITLE
feat: update social info on sign in

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Update social userinfo in `user.identities` each time a user is successfuly signed in by social connector.

## Linear Issue Reference

LOG-876

## Testing

before:

截屏2022-02-14 下午3 18 40

after:

image

### CODE REVIEW

**Review by xiaoyijun — APPROVED** [review]
LGTM

File: packages/core/src/lib/sign-in.ts [hunk 1]
```diff
@@ -136,7 +137,11 @@ export const signInWithSocial = async (
     })
   );
 
-  const { id } = await findUserByIdentity(connectorId, userInfo.id);
+  const { id, identities } = await findUserByIdentity(connectorId, userInfo.id);
+  // Update social connector's user info
+  await updateUserById(id, {
+    identities: { ...identities, [connectorId]: { userId: userInfo.id, details: userInfo } },
```
Comment (darcyYe) [inline packages/core/src/lib/sign-in.ts:143]: should we exclude `id` from `details`?

Comment (wangsijie) [reply packages/core/src/lib/sign-in.ts:143]: `details` is flexable and is ok to save reductent infos.

**Review by darcyYe — APPROVED** [review]
lgtm

**Pull request merged**